packages/blocks-ui/src/panels/property-controls.js

```diff
@@ -75,6 +77,42 @@ export default ({
                   ))}
                 </Field>
               )
+            } else if (value.type === ControlType.Boolean) {
+              return (
+                <div key={key}>
+                  <Label>{title}</Label>
+                  <Box
+                    sx={{
+                      display: 'flex',
+                      flexDirection: 'row',
+                      alignItems: 'center'
+                    }}
+                  >
+                    <Label sx={{ alignItems: 'center' }}>
+                      <Radio
```

Comment:
Sounds good to me

Reply:
So I just tried using Checkbox. It works differently with other inputs. We need to check `e.target.checked` for the value instead of `e.target.value`. To avoid making too many changes, I'll leave it as Radios for now.